### PR TITLE
feat(docs): add hints to set up https

### PR DESCRIPTION
**Pull request by BrocksiNet**
### Description

Add docs how to use https for localhost.

### Type of change

Docs update.

### CODE REVIEW

**Review by mdanilowicz — APPROVED** [review]

**Comment by patzick** [comment]
@BrocksiNet wouldn't be easier to setup it with built in https://vitejs.dev/config/server-options.html#server-https?
Docs says that using vite plugin https://github.com/vitejs/vite-plugin-basic-ssl 

I haven't tried it, but it looks good

**Comment by BrocksiNet** [comment]
I can try and update docs if easier. :+1:

File: apps/docs/src/resources/troubleshooting.md [hunk 1]
```diff
@@ -15,10 +15,43 @@ head:
...
+- Make sure you have `mkcert` installed on your system. Otherwise, follow [here](https://github.com/FiloSottile/mkcert) to set it up.
+- Create a valid certificate in your project folder by running `mkcert localhost`.
+- Update the `nuxt dev` command in your `package.json`.  
+  It should look like this: `NODE_TLS_REJECT_UNAUTHORIZED=0 nuxt dev --https --ssl-cert localhost.pem --ssl-key localhost-key.pem`
+- Now run your project with `npm run dev` or `pnpm run dev` from your project root.
+- Your browser may ask you to accept the risk when you visit https://localhost:3000. This is because it is a self-signed certificate.
+
+### Option 2: Vite plugin
+
+- Execute `npm add -D @vitejs/plugin-basic-ssl` in your project folder
```
Comment (patzick) [inline apps/docs/src/resources/troubleshooting.md:35]:
```suggestion
- Execute `pnpm add -D @vitejs/plugin-basic-ssl` in your project folder
```

File: apps/docs/src/resources/troubleshooting.md [hunk 2]
```diff
@@ -15,10 +15,43 @@ head:
...
+    https: true,
+  },
+  vite: {
+    plugins: [
+      basicSsl(),
+    ],
+  },
+  // ...
+  ```
+- Start your dev server with `npm run dev`
```
Comment (patzick) [inline apps/docs/src/resources/troubleshooting.md:52]:
```suggestion
- Start your dev server with `pnpm run dev`
```

**Review by patzick — APPROVED** [review]

**Pull request merged**